README.md

```diff
@@ -20,6 +20,16 @@ Some pages will need data locally to display, whitehall uses mySQL. You'll need
 
 - Once completed [a guide to install local data on whitehall can be found here](https://github.com/alphagov/govuk-docker/blob/master/docs/how-tos.md#how-to-replicate-data-locally)
 
+## Testing on intergration
```

Comment:
This is useful information but it's not part of removing the old files so it should go in separately. Also, I think it's best to add the instructions on deploying a branch to integration to the tech docs, assuming they aren't already there, because they could be useful for other apps/reasons. From Whitehall I'd reduce this down to a line about some pages not rendering locally, eg "Some pages do not render in the dev environment, in which case it is best to deploy changes to view and test on Integration (link to tech docs)"

Reply:
Salient point, this should live in tech docs. I've dropped this from this PR to update separately.